config/locales/idv/en.yml

```diff
@@ -132,21 +132,27 @@ en:
       sessions:
         no_pii: Do not use real personal information (demo purposes only)
     modal:
+      attempts:
+        one: 1 attempt
+        other: "%{count} attempts"
       button:
         warning: Try again
       finance:
         warning_accent: We could not find records matching your financial information.
         warning_html: >
-          %{accent} <span>Please check the information you entered.</span>
+          %{accent} <span>Please check the information you entered.</span> <span>
```

Comment:
Agreed.  I was going to do that in another PR, but since I still have some tweaking left of this one I will rework that portion as well.